Makefile

```diff
@@ -58,10 +58,8 @@ type-check:
 
 .PHONY: doclinter
 doclinter:
-	@sphinx-lint --enable all --max-line-length 85 --disable triple-backticks \
-	             -i CHANGES.rst -i LICENSE.rst -i EXAMPLES.rst \
-	             $(addprefix -i doc/, _build _static _templates _themes) \
```

Comment:
We really don't want those to be checked IMO.

Reply:
Would `-i doc/_*` achieve the same result?

Reply:
As with my other comment, should we then change this for the CI as well?

Reply:
I think the order-of-operations should be:

1. Fix all known issues in the `.rst` files in one PR (`all` enabled, `max-line-length` 85, no disabling of backticks).
1. Update the `Makefile` so that it produces sorted output with the above configuration.
1. Update CI to run `make doclinter` so that both are consistent.

Reply:
> no disabling of backticks

This must be disabled because otherwise we cannot have a literal rendering of something with triple backsticks. The triple backsticks *is* correct.

Reply:
all done